template/node/src/rpc.rs

```diff
@@ -137,7 +138,7 @@ where
 		client.clone(),
 		pool.clone(),
 		graph,
-		frontier_template_runtime::TransactionConverter,
+		Some(frontier_template_runtime::TransactionConverter),
```

Comment:
In our fork this value should be None

Reply:
This is the Frontier template, is not used in moonbeam

Reply:
Sorry my bad